pants.toml

```diff
@@ -22,18 +22,29 @@ backend_packages2 = [
 # List v2 plugins here.
 plugins2 = []
 
-
 [source]
 # The Python source root is the repo root. See https://pants.readme.io/docs/source-roots.
 source_roots = """{
   '': ('python',),
 }"""
 
-
 [python-setup]
 # The default interpreter compatibility for code in this repo. Individual targets can ovverride
 #  this with the `compatibility` field. See
 #  https://pants.readme.io/docs/python-interpreter-compatibility.
 interpreter_constraints = [">=3.7"]
 # Use a lockfile. See https://pants.readme.io/docs/python-third-party-dependencies.
 requirement_constraints = "constraints.txt"
+
+[lint]
+# We use `per_target_caching` because we have some Python 2-only targets and some Python 3-only
+# targets, so we need to run each target as a separate process to avoid interpreter compatibility
+# conflicts. If you only have one Python version in your repository, do not use this setting. See
+# https://pants.readme.io/docs/python-lint-goal.
+per_target_caching = true
```

Comment:
Isn't this the default? Should probably remove this option...

Reply:
No. This is very intentionally not the default because performance is so bad when this is true. And because this results in really verbose output, like 6 distinct Black results.

There are a few cases where you have a very warm cache that performance is better, but on average, the default to `false` is a good one.